thanos/plugindefinition.yaml

```diff
@@ -74,6 +74,13 @@ spec:
       name: thanos.query.serviceLabels
       required: false
       type: map
+    - default:
+        greenhouse.sap/expose: "true"
+        greenhouse.sap/exposed-named-port: "http"
+      description: Needed for exposing Thanos Query to the service proxy. UI is accessible via http, exposing this particular port.
+      name: thanos.query.serviceAnnotations
+      required: false
+      type: map
```

Comment:
Whenever you do any changes in the plugin, please bump the [PluginDefinition version](https://github.com/cloudoperators/greenhouse-extensions/blob/e53535a0a9a308eada87bbd45846ac2723026da8/thanos/plugindefinition.yaml#L9) and also the [helm chart version](https://github.com/cloudoperators/greenhouse-extensions/blob/e53535a0a9a308eada87bbd45846ac2723026da8/thanos/plugindefinition.yaml#L14).